scripts/build-ios-simulator.sh

```diff
@@ -2,6 +2,8 @@
 
 APP_NAME="Woof"
 xcodebuild -project Woof/${APP_NAME}.xcodeproj \
-           -target ${APP_NAME} \
+           -scheme NetworkService
+xcodebuild -project Woof/${APP_NAME}.xcodeproj \
+           -scheme ${APP_NAME} \
            -configuration Release \
-           -sdk iphonesimulator16.2
+           -sdk iphonesimulator16.4
```

Comment:
nit: you can extract the `16.4` suffix into the constant and put in the beginning of the script so everyone can easily edit them if it's required.

something like:
```bash
IOS_SDK="16.4"
```

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/commit/e65e3269a0ac9e61c73db68409846e8f4db93658